src/cli/cli.go

```diff
@@ -65,6 +65,8 @@ func preRun(cc *cobra.Command, args []string) error {
 		"Initialize a repository.",
 		"Initialize a S3 repository",
 		"Connect to a S3 repository",
+		"Initialize a repository on local or network storage.",
```

Comment:
we should probably make a const out of these one day.  Not an issue in this PR, just thinking out loud.